### PR TITLE
PE-38 | Implemented General and Payment pages

### PR DESCRIPTION
**Pull request by MaxLian11**
WHAT I DID:

- Implemented General settings page. It has 3 setting switches and an About button that shows some information about Eureka.
- Implemented Payment settings page. It has 2 options: Google pay and Apple pay. Pages redirect users to the blank pages in which specific settings will be implemented in the next task. Blank pages have confirmation buttons that represent successfully saved data in the application.

HOW TO TEST

- Make sure to `flutter pub get`
- Settings must initially be switched off, but once changed, it must store persistent data and show the last changes after leaving the page or after closing the application.
Screenshot_3
- About page must appear on top of the settings page without overflowing and being able to close.
- Google Pay and Apple pay have logos and those must be pulled from Assets without errors or delays.
Screenshot_4
- Confirmation message must redirect user back to settings without being able to go back to Google Pay or Apple Pay after confirmation alert.

### CODE REVIEW

File: lib/screens/settings/settings_payment.dart [hunk 1]
```diff
@@ -1,20 +1,44 @@
...
-      body: Center(
-        child: ElevatedButton(
-          onPressed: () {
-            Navigator.pop(context);
-          },
-          child: Text('Go back!'),
-        ),
+      body: ListView(
+        children: <Widget>[
+          Divider(height: 20.0),
```
Comment (ayeejane) [inline lib/screens/settings/settings_payment.dart]: you don't need this divider

File: lib/screens/settings/settings_payment.dart [hunk 2]
```diff
@@ -1,20 +1,44 @@
...
+          Text('Payment Methods', style: TextStyle(color: Colors.blue, fontSize: 30.0, fontWeight: FontWeight.bold), textAlign: TextAlign.center,),
```
Comment (ayeejane) [inline lib/screens/settings/settings_payment.dart]: make font smaller and align to the left instead of center

**Review by ayeejane — CHANGES_REQUESTED** [review]

File: lib/screens/settings/settings_payment.dart (hunk 1 above)
Comment (MaxLian11) [reply lib/screens/settings/settings_payment.dart]: I made it to move the page down a little bit. Do you think it is bad if I used divider for that?

Comment (MaxLian11) [reply lib/screens/settings/settings_payment.dart]: Ok, I figured it out

File: lib/screens/settings/settings_payment.dart [hunk 3]
```diff
@@ -1,20 +1,44 @@
...
+        ],
       ),
     );
   }
 }
+
+ListTile settingsListTile(
+    BuildContext context, AssetImage image, String string, Widget newScreen) {
+  return ListTile(
+    leading: Container(padding: EdgeInsets.all(10.0), child: Image(image: image, width: 100.0), decoration: BoxDecoration(border: Border.all())),
```
Comment (ayeejane) [inline lib/screens/settings/settings_payment.dart]: make icons smaller

**Review by ayeejane — CHANGES_REQUESTED** [review]
Everything works as intended! Just made a few comments on UI changes.

**Review by tonynguyen98 — APPROVED** [review]
Great work with this!

**Review by ayeejane — APPROVED** [review]
Looks good! :)

**Pull request merged**